### ISSUE TITLE
Cartesian DataHover

### ISSUE THREAD

**Issue by jddk1** [Live-Charts/Live-Charts#872]
Hi!

I´m using the LiveCharts API on a UWP application, however, when trying to add the event "DataHover" to a "CartesianChart", the Visual Studio IDE says me the following error:

The member "DataHover" is not recognized or is not accessible.
Note: I have the LiveCharts UWP version 0.8.0, and LiveCharts 0.9.7 installed by NuGet. I checked and it does not have any new version of LiveCharts.UWP to update.

There is any LiveChart version I should have, in order to have access to this event?

Best Regards

**Comment by diegocorradini** [comment]
I am developing a WPF application and I can add DataHover event to CartesianChart.

**Comment by jddk1** [comment]
> I am developing a WPF application and I can add DataHover event to CartesianChart.

Yes if I set my ".xaml" file to use "CartesianChart" from WPF library, I can access the "DataHover" event and add it to the Chart.

However since the library, I'm using its the UWP, that event is currently unavailable. I would like to know if there is a way to get access to this event, or if there is a similar event to this one that could do the trick.

**Comment by diegocorradini** [comment]
Sorry but I have no experience with UWP.
